src/simulators/statevector/qubitvector_thrust.hpp

```diff
@@ -799,8 +799,8 @@ void QubitVectorChunkContainer<data_t>::SetupP2P(int nDev)
       for(i=0;i<nDev;i++){
         if(i != m_iDevice){
           cudaDeviceCanAccessPeer(&m_p2pEnable[i],m_iDevice,i);
-          if(m_p2pEnable[i])
-            cudaDeviceEnablePeerAccess(i,0);
+//          if(m_p2pEnable[i])
```

Comment:
What happened here?... can't see the relation.
If we really want to disable this, just remove the lines.

Reply:
@doichanj fixed and checked the CUDA compilation on PPC and he found there were some problems with the tests because of these lines, so he disabled them.